File: go/go-fiber-postgres/internal/models/favorite.go
package models

import "time"

type Favorite struct {
	ID        int       `db:"id" json:"id"`
	UserID    int       `db:"user_id" json:"user_id"`
	ArticleID int       `db:"article_id" json:"article_id"`
	CreatedAt time.Time `db:"created_at" json:"created_at"`
}

File: csharp/dotnet-sqlserver/src/Api/Data/Entities/Favorite.cs
namespace Api.Data.Entities;

public class Favorite
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public User User { get; set; } = null!;
    public int ArticleId { get; set; }
    public Article Article { get; set; } = null!;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}
